### PR TITLE
Adedd task

### PR DESCRIPTION
**Pull request by GurgenBaghdasaryan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
-function range(x, y) {
-    let arr = [];
+let calculator = {
+    read(number1, number2) {
+        this.Number1 = number1;
+        this.Number2 = number2;
+        console.log(this);
```
Comment (arayik-99) [inline index.js]: Remove this console.log

File: index.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+    },
 
-    for (let i = x; i <= y; i++) {
-        arr[i - x] = i;
-    };
+    sum() {
+        this.result = this.Number1 + this.Number2;
```
Comment (arayik-99) [inline index.js]: Make this variables (remove this)

File: index.js [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+        return this.result;
+    },
 
-    return arr;
-};
+    mul() {
+        this.resultMul = this.Number1 * this.Number2;
```
Comment (arayik-99) [inline index.js]: Make this variables (remove this)

**Comment by edotnet** [comment]
why your commit is named `...`

File: index.js [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
+        this.Number1 = number1;
+        this.Number2 = number2;
+        return this;
+    },
 
-    for (let i = x; i <= y; i++) {
-        arr[i - x] = i;
-    };
+    sum() {
+        return resultSum = this.Number1 + this.Number2;
```
Comment (edotnet) [inline index.js]: `resultSum =` is not needed

File: index.js [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
+    },
 
-    return arr;
-};
+    mul() {
+        return  resultMul = this.Number1 * this.Number2;
```
Comment (edotnet) [inline index.js]: `resultMul =` is not needed

File: index.js [hunk 6]
```diff
@@ -1,11 +1,19 @@
-function range(x, y) {
-    let arr = [];
+let calculator = {
+    read(number1, number2) {
+        this.Number1 = number1;
+        this.Number2 = number2;
+        return this;
```
Comment (edotnet) [inline index.js]: Remove `return` statement

**Review by edotnet — CHANGES_REQUESTED** [review]

**Review by edotnet — APPROVED** [review]

File: index.js [hunk 7]
```diff
@@ -2,18 +2,17 @@ let calculator = {
     read(number1, number2) {
         this.Number1 = number1;
```
Comment (arayik-99) [inline index.js]: I would suggest to make the Number1 to number1 (same applies for number2), just to follow the naming convention.

**Review by arayik-99 — APPROVED** [review]

**Pull request merged**